packages/twenty-front/src/modules/settings/admin-panel/hooks/useFeatureFlagsManagement.ts

```diff
@@ -0,0 +1,100 @@
+import { UserLookup } from '@/settings/admin-panel/types/UserLookup';
+import { useState } from 'react';
+import { isDefined } from 'twenty-ui';
+import {
+  useUpdateWorkspaceFeatureFlagMutation,
+  useUserLookupAdminPanelMutation,
+} from '~/generated/graphql';
+
+export const useFeatureFlagsManagement = () => {
+  const [userLookupResult, setUserLookupResult] = useState<UserLookup | null>(
+    null,
+  );
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+
+  const [userLookup] = useUserLookupAdminPanelMutation({
+    onCompleted: (data) => {
+      setIsLoading(false);
+      if (isDefined(data?.userLookupAdminPanel)) {
+        setUserLookupResult(data.userLookupAdminPanel);
+      }
+    },
+    onError: (error) => {
+      setIsLoading(false);
+      setError(error.message);
+    },
+  });
+
+  const [updateFeatureFlag] = useUpdateWorkspaceFeatureFlagMutation({
+    onError: (error) => {
+      setError(error.message);
+    },
+  });
+
+  const handleUserLookup = async (userIdentifier: string) => {
+    setError(null);
+    setIsLoading(true);
+
+    try {
+      const response = await userLookup({
+        variables: { userIdentifier },
+      });
+
+      return response.data?.userLookupAdminPanel;
+    } catch {
+      return null;
```

Comment:
Why do we catch the error here / what are the different scenarios?  onError should catch the errors?

Reply:
This is an oversight :(